components/Workspaces/TrackedContributorsWizard/PasteContributorsStep.tsx

```diff
@@ -1,19 +1,19 @@
-import { useState } from "react";
+import { useLocalStorage } from "react-use";
 import Button from "components/atoms/Button/button";
 
 interface PasteContributorsStepProps {
   onBulkAddContributors: (contributors: string[]) => void;
 }
 
 export const PasteContributorsStep = ({ onBulkAddContributors }: PasteContributorsStepProps) => {
-  const [pastedInput, setPastedInput] = useState("");
+  const [pastedInput, setPastedInput] = useLocalStorage("bulk-add-contributors", "");
```

Comment:
refactor (blocking): Add a default value for pastedInput so the TypeScript bang operator (!) is not required in the two spots below.

```suggestion
  const [pastedInput = "", setPastedInput] = useLocalStorage("bulk-add-contributors", "");
```